lib/stupidedi/versions/functional_groups/004010/segment_defs/TD5.rb

```diff
@@ -11,9 +11,10 @@ module SegmentDefs
           TD5 = s::SegmentDef.build(:TD5, "Carrier Details (Routing Sequence/Transit Time)",
             "To specify the carrier and sequence of routing and provide transit time information",
             e::E133.simple_use(r::Optional,  s::RepeatCount.bounded(1)),
-            e::E66 .simple_use(r::Mandatory,  s::RepeatCount.bounded(1)),
+            e::E66 .simple_use(r::Relational, s::RepeatCount.bounded(1)),
```

Comment:
Relational implies there's a constraint that involves this element and another, like "this must be present if TDS-03 is present". One example of that is `N1`. If you don't see the constraint in your documentation, this should probably be made `Optional`.